lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,16 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      LGTM_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
+
+      def self.num_lgtm
+        "#{GitReflow::Config.get('constants.numlgtm')}".length > 0 ? "#{GitReflow::Config.get('constants.numlgtm')}" : ""
+      end
+
+      def self.lgtm_regex
+        "#{GitReflow::Config.get('constants.lgtmregex')}".length > 0 ? Regexp.new("#{GitReflow::Config.get('constants.lgtmregex')}") : LGTM_REGEX
```

Comment:
As a rule of thumb, anytime you need to horizontally scroll on Github during a review is a good time to break it up.  Normally I agree with you, but I think with all the string escapes it'd be easier to read each on its own line.